src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -9,18 +9,34 @@ namespace System.Diagnostics {
   public partial class Activity {
     public Activity(string operationName) {}      
     public string OperationName { get { throw null; } }
+
+    [System.ObsoleteAttribute]
```

Comment:
Seems like the concept of an Id is still useful (indeed see my comment on W3CId below).   I don't think we should mark it obsolete.

Reply:
By the way, I don't view Id as being required to match the hierarchical syntax we gave it.   Indeed the | was supposed to be a 'marker' that it WAS in that format.  Thus Id can be the W3C format (or other things).  It is just required to be unique.